site/_apps/zuix-hn.txt

```diff
@@ -0,0 +1,26 @@
+---
+weight: 2
```

Comment:
Didn't really know what was that for. I did copy&paste and left it as is. Of course it is ok.